google/cloud/storage/_helpers.py

```diff
@@ -563,3 +570,37 @@ def _bucket_bound_hostname_url(host, scheme=None):
         return host
 
     return "{scheme}://{host}/".format(scheme=scheme, host=host)
+
+
+def _api_core_retry_to_resumable_media_retry(retry, num_retries=None):
+    """Convert google.api.core.Retry to google.resumable_media.RetryStrategy.
+
+    Custom predicates are not translated.
```

Comment:
Is this clear to the end user?

Reply:
I'll have this warning in every single public method docstring, so with that it should be clear.